RELEASE.md

```diff
@@ -1,13 +1,5 @@
 # Current version (not yet released; still in development)
```

Comment:
drop you need to drop the `# Current version (not yet released; still in development)` too?